docs/readme.md

```diff
@@ -0,0 +1,154 @@
+# NGINX Gateway Fabric Docs
+
+This directory contains the user documentation for NGINX Gateway Fabric and the requirements for linting, building, and publishing the docs.
+
+We use [Hugo](https://gohugo.io/) to build the docs for NGINX, with the [nginx-hugo-theme](https://github.com/nginxinc/nginx-hugo-theme).
+
+Docs should be written in Markdown.
+
+In this directory, you will find the following files:
+
+- a [Netlify](https://netlify.com) configuration file;
+- configuration files for [markdownlint](https://github.com/DavidAnson/markdownlint/) and [markdown-link-check](https://github.com/tcort/markdown-link-check)
+- a `./config` directory that contains the [Hugo](https://gohugo.io) configuration.
+
+## Git workflow
+
+To contribute to the documentation, create your work branch from the `main` branch. Once you have added your changes, open a PR to merge your changes back to `main`.
```

Comment:
We use a forking workflow - see https://github.com/nginxinc/nginx-gateway-fabric/blob/main/docs/developer/branching-and-workflow.md